### ISSUE TITLE
Dict transformer doesn't match values

### ISSUE THREAD

**Issue by dani** [GreenmaskIO/greenmask#76]
In a database, I'd like to transform values of a column using the Dict transformer. The original database has this:

![image](https://github.com/GreenmaskIO/greenmask/assets/526826/ebd24bd5-498b-4d14-8788-2d40af6ae35c)

The transformer used is configured like this
```yaml
  transformation:

    - schema: public
      name: provider
      transformers:
        - name: Dict
          params:
            column: name
            values:
              Clinique Louis Pasteur Nancy: "Établissement 1"
              Clinique Ambroise Paré Thionville: "Établissement 2"
              Polyclinique La Ligne bleue: "Établissement 3"
              Clinique Jeanne d'Arc: "Établissement 4"
            # fail_not_matched: false
```

Yet, when validating or dumping, greenmask fails with

```
2024-04-25T16:06:46+02:00 WRN error flushing gzip buffer error="io: read/write on closed pipe"
2024-04-25T16:06:46+02:00 WRN error closing TableDumper writer error="error closing gzip writer: io: read/write on closed pipe"
2024-04-25T16:06:46+02:00 WRN error flushing gzip buffer error="io: read/write on closed pipe"
2024-04-25T16:06:46+02:00 WRN error closing TableDumper writer error="error closing gzip writer: io: read/write on closed pipe"
2024-04-25T16:06:46+02:00 WRN error flushing gzip buffer error="io: read/write on closed pipe"
2024-04-25T16:06:46+02:00 WRN error closing TableDumper writer error="error closing gzip writer: io: read/write on closed pipe"
2024-04-25T16:06:46+02:00 FTL cannot make a backup error="data stage dumping error: at least one worker exited with error: error processing table dump: dump error: dump error on table public.provider at line 1: dump error on table public.provider at line 1: unable to match value for \"Polyclinique La Ligne bleue\""
```

I tried to quote (single and double) the keys in greemask config, with no difference. I even tried with simple keys (without space or special chars) with the same result.

**Comment by wwoytenko** [comment]
Hi! Thank you for reporting. I will check it ASAP.

**Comment by wwoytenko** [comment]
I've just checked. The greenmask applied lowercase somehow. This is a bug.

![image](https://github.com/GreenmaskIO/greenmask/assets/74241416/7b72ec9e-6acb-4cfb-acea-ce517aa3a53f)

**Comment by wwoytenko** [comment]
It is not a bug, it is likely a design. So I should consider the best way how to overcome this obstacle

**Comment by wwoytenko** [comment]
@dani 

I've made a PR with the workaround. I need some time to test it. If everything is alright I will publish it in the next release (next week)

**Comment by wwoytenko** [comment]
Fixed. You can build the main branch manually by calling `make build` or wait for the next release (expected next week)

**Comment by wwoytenko** [comment]
Published in v0.1.11
